apollo-router/src/plugins/telemetry/metrics/otlp.rs

```diff
@@ -41,8 +42,33 @@ impl MetricsConfigurator for super::super::otlp::Config {
             Some(exporter) => {
                 let exporter = opentelemetry_otlp::new_pipeline()
                     .metrics(
```

Comment:
Might make sense to add a comment for this, as to why `inexpensive()` selector is not used.